### ISSUE TITLE
Show warning if ledger is using older stx app

### ISSUE THREAD

**Issue by aulneau** [blockstack/stacks-wallet#722]
I was trying to sign some transactions recently and I had not updated my STX app on my ledger. I would be able to auth into the Wallet just fine, however when it came to receiving the signed payload from the ledger, it would simply stall at the modal and never resolve. Once I updated the STX app on my ledger, the signed transaction worked as expected. Is there an ability to know the version of the stx app the connected ledger is running? If so, we should pop a toast or notice or something like that and let folks know to update.

**Comment by kyranjamie** [comment]
Good idea. Yeah there is, the device reports it. We can add a known-latest version constant, and prompt users to upgrade if they're below that.
